backend/etcd/etcd.go

```diff
@@ -31,6 +31,9 @@ const (
 	// EtcdStartupTimeout is the amount of time we give the embedded Etcd Server
 	// to start.
 	EtcdStartupTimeout = 60 // seconds
+
+	DefaultMaxRequestBytes   = 1.5 * (1 << 20)
```

Comment:
Would you oppose comments for these constants?

Reply:
Done.

Reply:
![giphy](https://user-images.githubusercontent.com/10634078/57946517-513c2d00-7891-11e9-8803-e91b4633a30c.gif)